src/sgame/sg_client.cpp

```diff
@@ -928,6 +928,13 @@ const char *ClientUserinfoChanged( int clientNum, bool forceName )
 			trap_SendServerCommand( ent - g_entities, va( "print_tr %s %s %s", QQ( "$1t$ $2$" ), Quote( err ), Quote( newname ) ) );
 			revertName = true;
 		}
+		else if ( Q_UTF8_Strlen( newname ) > MAX_NAME_CHARACTERS )
```

Comment:
Does this take colors into account?

Reply:
Colors do not count against you, however, emoticons do.

Reply:
Sounds good, thanks